src/utils/globalId.js

```diff
@@ -0,0 +1,10 @@
+import base64 from 'base-64';
+
+export const toGlobalId = ({ type, id }) =>
+  base64.encode(`${type}:${id}:${process.env.RBB_SECRET}`);
```

Comment:
@racedale Note we need to set some secret in prod- again, it can be whatever you like